docs/DeveloperGuide.md

```diff
@@ -205,28 +223,79 @@ The image below illustrates the class diagram in the context of data saving and
 ![img_3.png](DataManagerCD.drawio.png)
 
 
-`DataManager` has 2 main objectives which are to save data into csv files and load data from csv files. The sequence of
-saving and loading are the same for both `StonksXD_Entries.csv` and `StonksXD_Budget.csv`. Because of this, we will 
-only be showing sequence diagrams in the context of saving and loading entries.
+#### Saving
```

Comment:
Perhaps it is better for loading to come before saving since the program does loading before saving as well?